spec/lib/iiif_manifest/manifest_factory_spec.rb

```diff
@@ -163,6 +163,96 @@ def sequence_rendering
       end
     end
 
+    context 'where there is a no manifest_metadata method' do
+      let(:file_presenter) { DisplayImagePresenter.new }
+
+      it 'does not have a metadata element' do
+        allow(IIIFManifest::ManifestBuilder::CanvasBuilder).to receive(:new).and_call_original
+        allow(book_presenter).to receive(:file_set_presenters).and_return([file_presenter])
```

Comment:
You don't really need these two allows and the file_presenter let in order for this test to run successfully.